app/src/main/resources/io/apicurio/registry/storage/impl/sql/h2.ddl

```diff
@@ -6,31 +6,60 @@ CREATE TABLE apicurio (prop_name VARCHAR(255) NOT NULL, prop_value VARCHAR(255))
 ALTER TABLE apicurio ADD PRIMARY KEY (prop_name);
 INSERT INTO apicurio (prop_name, prop_value) VALUES ('db_version', 100);
 
-CREATE TABLE sequences (name VARCHAR(32) NOT NULL, seq_value BIGINT NOT NULL);
-ALTER TABLE sequences ADD PRIMARY KEY (name);
+CREATE TABLE sequences (seq_name VARCHAR(32) NOT NULL, seq_value BIGINT NOT NULL);
+ALTER TABLE sequences ADD PRIMARY KEY (seq_name);
 
-CREATE TABLE globalrules (type VARCHAR(32) NOT NULL, configuration TEXT NOT NULL);
-ALTER TABLE globalrules ADD PRIMARY KEY (type);
+CREATE TABLE config (pname VARCHAR(255) NOT NULL, pvalue VARCHAR(1024) NOT NULL, modifiedOn BIGINT NOT NULL);
```

Comment:
Since we're doing some renames, can we unify `seq_name` and `pname`, and `prop_name`?

Reply:
and `pkey` in `group_labels` below.

Reply:
I like `{table}_key` and `{table}_value` but don't care much.

Reply:
Yeah that makes sense.  I'll do it.

Reply:
I'm using `propName`, `propValue`, `labelKey`, and `labelValue`.  Camelcase for column names is more consistent across the rest of the ddl.  We're using underscores for table names, and camelcase for column names.  Weird but ok I think.